lib/internal/modules/cjs/loader.js

```diff
@@ -162,6 +162,7 @@ function stat(filename) {
   }
   return result;
 }
+Module._stat = stat;
```

Comment:
Since this is meant only for experimentation, maybe it would be worth to add a experimental warning todiscourage its use in production:
```suggestion
let _stat = stat;
ObjectDefineProperty(Module, '_stat', {
  __proto__: null,
  get() { return _stat; },
  set(stat) {
    emitExperimentalWarning('…');
    _stat = stat;
    return true;
  },
  configurable: true,
});
```